src/cmds/scripts/pbs_init.d.in

```diff
@@ -113,22 +113,21 @@ lc_host_name()
 #
 : check_started
 check_started() {
+	ps_out=`ps -p $1 -o args 2> /dev/null | tail -1`
+	if [ -z "${ps_out}" -o "`echo ${ps_out} | cut -c1`" = "[" ]; then
+		ps_out=`ps -p $1 -o command 2> /dev/null | tail -1`
+		if [ -z "${ps_out}" ]; then
+			return 1
+		fi
+	fi
 
-  ps_out=`ps -p $1 -o args 2> /dev/null | tail -1`
-  if [ -z "${ps_out}" -o "`echo ${ps_out} | cut -c1`" = "[" ]  ; then
-    ps_out=`ps -p $1 -o command 2> /dev/null | tail -1`
-    if [ -z "${ps_out}" ]; then
-	return 1
-    fi
-  fi
-
-# strip out everything except executable name
-  prog_name=`echo ${ps_out} | grep -how "$2"`
-  if [ "x${prog_name}" = "x$2" ] ; then
-    return 0;
-  fi
+	# strip out everything except executable name
+	prog_name=`echo ${ps_out} | grep -how "$2"`
+	if [ "x${prog_name}" = "x$2" ]; then
```

Comment:
not sure, but shouldn't this be `==`?

Reply:
it seems like == is a bash specific operator which is an alias of =, = is more portable across different shells, so we should use =:
https://stackoverflow.com/questions/20449543/bash-equality-operators-eq

Reply:
I don't think so @agrawalravi90, we use /bin/bash as shebang line in this file.

Reply:
= and == both work for string matching.